astroquery/mast/core.py

```diff
@@ -290,6 +297,108 @@ class ObservationsClass(MastClass):
     Class for querying MAST observational data.
     """
 
+    def __init__(self):
+
+        super(ObservationsClass, self).__init__()
+
+        self._caomCols = None  # Hold Mast.Caom.Cone columns config
+
+    def _get_caom_col_config(self):
+        """
+        Gets the columnsConfig entry for Mast.Caom.Cone and stores it in self.caomCols.
+        """
+
+        headers = {"User-Agent": self._session.headers["User-Agent"],
+                   "Content-type": "application/x-www-form-urlencoded",
+                   "Accept": "text/plain"}
+
+        response = Mast._request("POST", self._SERVER+"/portal/Mashup/Mashup.asmx/columnsconfig",
```

Comment:
I like it in some of the other modules when we collect all the urls at the top of the file (e.g. astroquery/sdss/core.py). It's much easier to update when need, etc.
However we don't do that (yet) in all the modules, so totally fine to leave it as is.